app/deepseg.cc

```diff
@@ -495,18 +529,21 @@ int main(int argc, char* argv[]) {
 		}
 
 		// timing details..
-		printf("wait:%9ld lock:%9ld [grab:%9ld retr:%9ld] copy:%9ld prep:%9ld tflt:%9ld mask:%9ld post:%9ld v4l2:%9ld FPS: %5.2f\e[K\r",
-			diffnanosecs(ti.waitns,ti.lastns),
-			diffnanosecs(ti.lockns,ti.waitns),
-			ti.grabns,
-			ti.retrns,
-			diffnanosecs(ti.copyns,ti.lockns),
+		printf("main [grab:%9ld retr:%9ld copy:%9ld prep:%9ld mask:%9ld post:%9ld v4l2:%9ld FPS: %5.2f] ai: [wait:%9ld prep:%9ld tflt:%9ld mask:%9ld FPS: %5.2f] \e[K\r",
+			diffnanosecs(ti.grabns, ti.lastns),
+			diffnanosecs(ti.retrns,ti.grabns),
+			diffnanosecs(ti.copyns,ti.retrns),
 			diffnanosecs(ti.prepns,ti.copyns),
-			diffnanosecs(ti.tfltns,ti.prepns),
-			diffnanosecs(ti.maskns,ti.tfltns),
+			diffnanosecs(ti.maskns,ti.prepns),
 			diffnanosecs(ti.postns,ti.maskns),
 			diffnanosecs(ti.v4l2ns,ti.postns),
-			1e9/diffnanosecs(ti.v4l2ns,ti.lastns));
+			1e9/diffnanosecs(ti.v4l2ns,ti.lastns),
```

Comment:
Whitespace after comma should be homogeneous.

Reply:
Thanks, I missed that one.